WSFrontend/org.openl.rules.ruleservice/test/org/openl/rules/ruleservice/loader/DeployStrategyTest.java

```diff
@@ -0,0 +1,21 @@
+package org.openl.rules.ruleservice.loader;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class DeployStrategyTest {
+
+    @Test
+    public void testFromString() {
+        assertEquals(DeployStrategy.IF_ABSENT, DeployStrategy.fromString("IF_ABSENT"));
+        assertEquals(DeployStrategy.IF_ABSENT, DeployStrategy.fromString("if-absent"));
```

Comment:
I think we need to support only one variant - IF_ABSENT

Reply:
done